MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  File.swift
+//  MVP
+//
+//  Created by Fei Wen on 6/12/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+struct FeedErrorViewModel {
+	let isError: Bool
```

Comment:
The `isError: Bool` could be an optional string `message: String?`, so the Presenter can decide when to show/hide the message and which message to show.

```suggestion
	let message: String?
```

So, for example, in the `didStartLoadingFeed()` method, the Presenter can call  `feedErrorView.display(FeedErrorViewModel(message: .none))` to hide the error.

And on error, the Presenter can call  `feedErrorView.display(FeedErrorViewModel(message: Localized.Feed.loadError))` to show the formatted error.

This way, the ViewController doesn't need to check a boolean value and then show or hide the message, and the Presenter can decide to show different messages without having to alter the view controller or the view model. It's easier to manage presentation state transitions when all changes are centralized in the presenter.